miro_preprocessor/xml_to_json_run_task/src/conftest.py

```diff
@@ -0,0 +1,53 @@
+import boto3
```

Comment:
Not to be addressed right now, but we should really find a way to stop just copying these files around.